MetaTempUnit/Hardware/library/KelvinVR/src/KelvinVR.h

```diff
@@ -54,7 +54,7 @@ class Kelvin
 public:
 	Kelvin();
 	Kelvin(const String &modelName);
-	void begin();
+	void begin(int bitRate = 9600);
```

Comment:
Add default speed to README file (Features description & prototype data)

Reply:
Added README for library + comments deleted